components/common/WalletModal.vue

```diff
@@ -109,6 +119,7 @@ export default class WalletModal extends Vue {
   protected guideUrl = ''
   protected extensionUrl = ''
   protected walletAccounts: WalletAccount[] = []
+  private hasUserAuthorized = false
```

Comment:
Can we store it in localStorage? same thing we do for Cookies
Since everytime I reload the page it ask me again to check the checkbox